src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -246,6 +247,27 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
     experimentType: ExperimentType.CLIENT_SIDE,
   });
 
+  // EXPERIMENT: Personalised Content Rail
+  const personalisedContentExperimentName = 'newswb_ws_location_based_topics';
+  let personalisedContentExperimentVariant = useOptimizelyVariation({
+    experimentName: personalisedContentExperimentName,
+    experimentType: ExperimentType.CLIENT_SIDE,
+  });
+
+  // temp overrides for testing
+  const personalisedContentOverride = true;
+  personalisedContentExperimentVariant = 'personalised';
```

Comment:
These overrides have been left in so that the PR can be checked locally by reviewers.

The logic will be replaced with `variation 1` and `variation 2` logic in https://bbc.atlassian.net/jira/software/c/projects/WS/boards/1251?selectedIssue=WS-1688 next stage

Reply:
Yeah already have